tasks/subscription-manager.yml

```diff
@@ -47,6 +47,38 @@
            and __rhc_subman_identity.rc == 1
            and rhc_release != __rhc_state_absent )
       else omit }}"
+    server_proxy_hostname: "{% if rhc_proxy is none -%}
+      {{ omit }}
+      {%- elif rhc_proxy == {} -%}
```

Comment:
you have an empty `elif` clause here?

Reply:
it's on purpose: if the dictionary is empty, it means "reset every proxy attribute to their default", and for all of the ones added the default is "empty string"

Reply:
No, I meant that you have an `elif` and there is nothing done - I've never seen this construct in jinja before - I guess it is an implicit no-op?  something like
```
elif rhc_proxy == {}:
  pass
elif ....
```

Reply:
The result is an empty string, which is what I want.